user-creation/src/test/java/no/unit/nva/useraccessservice/usercreation/CristinPersonRegistryTest.java

```diff
@@ -91,8 +94,10 @@ void shouldThrowExceptionIfCristinIsUnavailable(WireMockRuntimeInfo wireMockRunt
                                                                     ServiceConstants.API_DOMAIN,
                                                                     defaultRequestHeaders,
                                                                     new SecretsReader(secretsManagerClient));
-        var nin = NationalIdentityNumber.fromString(randomString());
-        assertThrows(PersonRegistryException.class, () -> personRegistry.fetchPersonByNin(nin));
+        var nin = NationalIdentityNumber.fromString(randomNin());
+        var exception = assertThrows(PersonRegistryException.class, () -> personRegistry.fetchPersonByNin(nin));
+        assertThat(exception.getMessage(), not(containsString(nin.toString())));
+        assertThat(appender.getMessages(), not(containsString(nin.toString())));
```

Comment:
This might not be necessary, but I'm trying to avoid further accidental mistakes